colorterms/Monster_to_SDSS_band_u.yaml

```diff
@@ -10,10 +12,10 @@ nodes:
 - 2.7222222222222223
 - 3.111111111111111
 - 3.5
-source_color_field_1: psfMag_g_flux
-source_color_field_2: psfMag_r_flux
-source_field: psfMag_u_flux
-source_survey: SDSS
+source_color_field_1: monster_DES_g_flux
+source_color_field_2: monster_DES_r_flux
+source_field: monster_SDSS_u_flux
+source_survey: Monster
```

Comment:
Hmm.  This is the one that makes me curious if it can work for the first pass as well as the second pass.

Reply:
Oh maybe we need Monster_to_SDSS_band_u.yaml and SDSS_to_SDSS_band_u.yaml files

Reply:
the isolate and transform step https://github.com/lsst-dm/the_monster/blob/24cc755ad5350beb89c1936a7fd17575b6b582c1/python/lsst/the/monster/isolate_and_transform.py#L109

does in fact require and SDSS_to_SDSS colorterm file (as well as DES_to_DES colorterm files).

My inclination is to include colorterm files that start with SDSS and DES as well as Monster .... do you have thoughts? I think the other option would be to do something clever in the cat_info.colorterm_file() method which I dont love.